README.md

```diff
@@ -1,40 +1,329 @@
-Welcome to _Dash Wallet_, a standalone Dash payment app for your Android device!
+### WOC Buy API Setup (Android)
 
-This project contains several sub-projects:
```

Comment:
This file should not be changed.  This is the main read me for the repo.  Wall Of Coins api specific information should go in a separate file.